app/views/migration/_form.html.erb

```diff
@@ -91,6 +91,6 @@
 
     <span class="required-fields-hint"><%= t('global.forms.hints.required-fields') %></span>
     <span class="error global hidden"><%= t('global.forms.validation-errors.form-has-errors') %></span>
-    <%= contribution.button(:submit, t('contribute.helpers.submit.contribution.create'), class: 'eu-blue') %>
+    <%= contribution.button(:submit, t('contribute.helpers.submit.contribution.create'), class: 'btn alt') %>
```

Comment:
You should be able to remove `btn` from here as it's included by default, and with this occurs twice.